release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -151,6 +152,14 @@ With this release, you can configure an {product-title} cluster on any on-premis
 
 For more information about this feature on bare-metal infrastructure, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#nw-osp-services-external-load-balancer_ipi-install-installation-workflow[Services for a user-managed load balancer] in _Setting up the environment for an OpenShift installation_.
 
+[id="ocp-4-16-detect-warning-for-iptables"]
+==== Detect and warning for iptables
+With this release, if you have pods in your cluster using `iptables` rules on either {op-system-base} or third-party software the following event message is given to warn against future deprecation:
+`This pod appears to have created one or more iptables rules. IPTables is deprecated and will no longer be available in RHEL 10 and later. You should consider migrating to another API such as nftables or eBPF.`
+
+
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_firewalls_and_packet_filters/getting-started-with-nftables_firewall-packet-filters#doc-wrapper[Getting started with nftables]. If you are running third-party software, check with your vendor ensure they will have an `nftables` based version available soon.
```

Comment:
missing a word or something ("TO ensure"?)

Reply:
I sure am. Thanks, Dan.